src/Graphics/src/Graphics/Platforms/Windows/PlatformStringSizeService.cs

```diff
@@ -28,7 +29,8 @@ public SizeF GetStringSize(string value, IFont font, float textSize, HorizontalA
 			{
 				FontFamily = font.Name,
 				FontSize = textSize,
-				FontWeight = new FontWeight { Weight = (ushort)font.Weight },
+				// Ensure font weight stays within the valid range (1–999) to avoid runtime errors
+				FontWeight = new FontWeight { Weight = (ushort)Math.Clamp(font.Weight, 1, 999) },
```

Comment:
We need the same approach in different classes. Can create an extension method and reuse it.

Reply:
should we add a extension method to parse this?